_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout:       post
+title:        Why WeWork.com uses a static generator and why you should too
+author:       ramin_bozorgzadeh
+summary:      Back when the web first started, things were a lot simpler. Most websites were made up of static html pages and there weren't a lot of moving parts. This is the story of how wework.com went from a monolithic "web app" to a statically generated site, and why ...
+image:        //res.cloudinary.com/wework/image/upload/c_fill,f_auto,g_faces,h_800,w_1000/v1448734984/engineering/wework-com-is-going-static.jpg
+categories:   engineering
+---
+
+### What is this "static" business all about?
+
+Over the past few years, static site generators have become a hot topic, so much so that [entire sites](https://www.staticgen.com/) have emerged dedicated to tracking and rating them. Don't take my word
+for it, check out this graph from [Google Trends for "static site generator"][2]:
+
+[![Google Trends - Static Site Generator][1]][2]
+[1]: http://res.cloudinary.com/wework/image/upload/f_auto/v1448735896/engineering/static-site-generator.png
+[2]: https://www.google.com/trends/explore#q=static%20site%20generator
+
+The idea is actually quite simple. On a traditional *dynamic* site, each page is processed and generated on the server per request (let's ignore caching strategies for the sake of argument.) A static site, on the other hand, is processed and generated just **once** on each deploy and all the server has to do is serve up the resulting generated HTML. Of course, this is an overly simplified explanation of how it all works, and this process doesn't work as well on all sites, but it does work really well for marketing and informational sites like [wework.com](https://www.wework.com). Not to mention that this very blog you are reading is a statically generated site using [Jekyll](http://jekyllrb.com/) and hosted on [Github pages](https://pages.github.com/)!
```

Comment:
we always trace it back in GA or Mixpanel via the referrer. But i doubt we'll get much.